subscriptions.go

```diff
@@ -28,8 +28,8 @@ type Subscriptions struct {
 
 // CancelSubscriptionParams represents arguments to be marshalled into JSON.
 type CancelSubscriptionParams struct {
-	CancelledAt       string   `json:"cancelled_at,omitempty"`
-	CancellationDates []string `json:"cancellation_dates,omitempty"`
+	CancelledAt       string    `json:"cancelled_at,omitempty"`
+	CancellationDates *[]string `json:"cancellation_dates,omitempty"`
```

Comment:
This is a breaking change, though. Release as 1.2.0.
Whoever uses this will now get compilation error.